src/mod_webdav.c

```diff
@@ -869,8 +870,7 @@ static void webdav_activelock(buffer *b,
 	buffer_append_int(b, timeout);
 	buffer_append_string_len(b, CONST_STR_LEN("</D:timeout>\n"));
 
-	buffer_append_string_len(b, CONST_STR_LEN("<D:owner>"));
-	buffer_append_string_len(b, CONST_STR_LEN("</D:owner>\n"));
+	buffer_append_string(b, owner);
```

Comment:
This looks incorrect.  Shouldn't the addition of owner be inbetween the <D:owner>...</D:owner> tags instead of removing those?

Reply:
The format of owner in db is defined by https://github.com/lighttpd/lighttpd1.4/pull/97/files#diff-4c7084e5d651be053c7f6ac417f2fee0R2526 , with <D:owner>...</D:owner> wrap around the content of owner. This is because of the following example
```xml
<D:lockinfo xmlns:D='DAV:' xmlns:A='MY_NS'>
  <D:lockscope><D:exclusive/></D:lockscope>
  <D:locktype><D:write/></D:locktype>
  <D:owner>
    <D:href>http://example.org</D:href>
    <A:some-property>xxxx</A:some-property>
    plain text
  </D:owner>
</D:lockinfo>
```

There's another concern that values for "owner" are likely to be the same. Should they be stored in a separate table to reduce redundancy?